app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -845,8 +859,9 @@ def process_roles(roles, task)
     process_elements(roles, MiqUserRole, task)
   end
 
-  def process_tenants(tenants, task)
-    process_elements(tenants, Tenant, task, "Tenant", "name")
+  def process_tenants(tenants, task, display_model = 'Tenant')
```

Comment:
is the problem that `process_elements` is adding the flash in a way you don't like?